app/controllers/users_controller.rb

```diff
@@ -1,6 +1,9 @@
 class UsersController < ApplicationController
   def accept_transition_info
     User.member(current_user['user_id']).accept_transition_screen
+  rescue StandardError => e
```

Comment:
My only misgiving here is that any error here will turn into the user getting stuck on the `accept_transition_screen`. The only situation we know how to handle here is if the error has already accepted the transition screen, so can we find a more specific error to catch here? 

Seeing as this is an acute problem that we need to sort out sooner rather than later, I'd be happy if we used the `JsonApiClient::ClientError` or whatever we had before here until we workout a better way to do it.

Reply:
> My only misgiving here is that any error here will turn into the user getting stuck on the accept_transition_screen.

The user won't be stuck, there's an `ensure` to redirect them below. Am I missing something?

Reply:
And yeah, I can use `JsonApiClient::ClientError`, but I also want to catch `JsonApiClient::ServerError`, is this how that would work?

```
rescue JsonApiClient::ClientError => e
  Raven.capture_exception(e)
rescue JsonApiClient::ServerError => e
  Raven.capture_exception(e)
```

Reply:
I think it's possible to do

```ruby
rescue JsonApiClient::ClientError, JsonApiClient::ServerError => e
  Raven.capture_exception(e)
```

Reply:
> The user won't be stuck, there's an ensure to redirect them below. Am I missing something?

@tvararu I'm imagining the situation where another error occurs that's not "The user has already accepted this screen.". A problem with the token, the backend not being happy with something ... as unlikely as this would be, the impact is pretty high. The steps I think would happen:

1. The user is shown the transition page, clicks on accept.
2. An error other than "they have already accepted" is fired, the ensure redirects them to "/providers".
3. The app tries to load providers, but sees that they haven't seen the transition info page (it was blocked above by the error), it redirects them to the transition page.
4. Go back to 1.

I know it's unlikely, but the impact is quite high (as bad as if not worse than an error that they may be able to recover from by hitting another page), and it's also relatively easy to protect against.

Reply:
@misaka done